README.md

```diff
@@ -193,16 +193,16 @@ Linux,
 
 2. Once you have Postman open, click on file -> import and import the .json file
 3. Click on the settings gear icon on the far top right (next to the eye icon) and click to add a new environment.
-4. Name your environment `dev` and create a variable called `api_url`. Give it a value of `https://localhost:8000`, which is the URL of your Django dev environment when it is running.
-5. Now, as long you have the Django app (https://localhost:8000) running, you should be able to make requests like POST Create User and POST Authenticate.
+4. Name your environment `dev` and create a variable called `api_url`.  For both the "Initial Value" and "Current Value," fill in `http://localhost:8000` which is the URL of your Django dev environment when it is running.
+5. Now, as long you have the Django app (http://localhost:8000) running, you should be able to make requests like POST Create User and POST Authenticate.
    Click on this link to see what you should expect: https://imgur.com/hd9VB6k
 
 - `POST` Create User will create a new user in your `localhost:8000` running Django app,
 - making a request to `POST Authenticate` will authenticate whether or not that user exists.
 
 ![screenshot of Postman environment variable setup](https://i.imgur.com/6Uq9XQp.png)
```

Comment:
Let's also update this screenshot of the Postman environment variable setup to be this image:

https://user-images.githubusercontent.com/4512699/88751261-14fa9280-d10c-11ea-80c4-2934c27254ad.png

Reply:
Update screenshot for Postman configuration